buffer/src/main/java/io/netty/buffer/PooledByteBufAllocator.java

```diff
@@ -124,8 +124,15 @@ public void run() {
         DEFAULT_CACHE_TRIM_INTERVAL = SystemPropertyUtil.getInt(
                 "io.netty.allocator.cacheTrimInterval", 8192);
 
-        DEFAULT_CACHE_TRIM_INTERVAL_MILLIS = SystemPropertyUtil.getLong(
-                "io.netty.allocation.cacheTrimIntervalMillis", 0);
+        if (SystemPropertyUtil.contains("io.netty.allocation.cacheTrimIntervalMillis")) {
+            logger.warn("-Dio.netty.allocation.cacheTrimIntervalMillis is deprecated," +
+                    " use -Dio.netty.allocator.cacheTrimIntervalMillis");
+            DEFAULT_CACHE_TRIM_INTERVAL_MILLIS = SystemPropertyUtil.getLong(
+                    "io.netty.allocation.cacheTrimIntervalMillis", 0);
```

Comment:
In case where both are specified (I know unlikely) would it be better to warn but use the new one?

Reply:
@njhill yep... good point. Let me do this

Reply:
done... will merge once CI passes